### PR TITLE
Feature/tt 1153 import

### PR DESCRIPTION
**Pull request by sarahentzel**

### CODE REVIEW

File: src/store/importexport/actions.tsx [hunk 1]
```diff
@@ -49,3 +59,87 @@ export const exportProject = (
...
+        keyMap: memory.keyMap,
+      };
+      const ser = new JSONAPISerializerCustom(s);
+      ser.resourceKey = () => {
+        return 'remoteId';
+      };
+      files.forEach(function(file) {
+        var data = fs.readFileSync(filepath + 'data/' + file);
+        var json = ser.deserialize(
+          JSON.parse(data.toString()) as ResourceDocument
```
Comment (gtryus) [inline src/store/importexport/actions.tsx]: I like this simple solution 💯

File: src/store/importexport/reducers.tsx [hunk 2]
```diff
@@ -44,6 +48,30 @@ export default function(
...
+      return {
+        ...state,
+        importexportStatus: action.payload,
+      };
+    /*
+    case IMPORT_COMPLETE:
+      return {
+        ...state,
+        importexportStatus: action.payload,
+      }; */
```
Comment (gtryus) [inline src/store/importexport/reducers.tsx]: Did you want to delete these lines?

File: src/components/Transcriber.tsx [hunk 3]
```diff
@@ -603,7 +604,7 @@ export function Transcriber(props: IProps) {
             </Grid>
           </Grid>
           <Grid container direction="row" className={classes.row}>
-            <Grid container xs justify="center">
```
[the remark this answers is not in the document]
Comment (sarahentzel) [reply src/components/Transcriber.tsx]: console errors, which I was having to wade through to find my relevant console logs.  xs is just for items, not for containers

File: src/components/Transcriber.tsx [hunk 4]
```diff
@@ -735,7 +736,7 @@ export function Transcriber(props: IProps) {
                 label={t.makeComment}
               />
             </Grid>
-            <Grid container xs justify="flex-end">
```
[the remark this answers is not in the document]
Comment (sarahentzel) [reply src/components/Transcriber.tsx]: console errors, which I was having to wade through to find my relevant console logs. xs is just for items, not for containers

File: src/components/Transcriber.tsx [hunk 5]
```diff
@@ -784,14 +785,16 @@ export function Transcriber(props: IProps) {
...
                   </Button>
-                  <Button
-                    variant="contained"
-                    color="primary"
-                    className={classes.button}
-                    onClick={handleClose}
-                  >
-                    {t.close}
-                  </Button>
+                  {true && (
```
[the remark this answers is not in the document]
Comment (sarahentzel) [reply src/components/Transcriber.tsx]: hmm...I think that was leftover from testing something.  I'll remove it.

File: src/store/importexport/actions.tsx (hunk 1 above)
Comment (sarahentzel) [reply src/store/importexport/actions.tsx]: Yes, thank you for this suggestion!

File: src/components/Transcriber.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (gtryus) [reply src/components/Transcriber.tsx]: I resolved this by creating another level in the Grid formatting for the item with the xs. Without the xs, the buttons format on two lines and don't format the way we want.

File: src/components/Transcriber.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (gtryus) [reply src/components/Transcriber.tsx]: Similarly here, I added an extra level for the xs formatting. This allows the buttons on the bottom to appear on the same row as the make comment which is the formatting we want. Just removing the xs caused the buttons to appear on a separate line.

**Pull request merged**